src/BTR3.ohm

```diff
@@ -120,10 +122,17 @@ BTR3 {
     hh = ("0".."1" digit) | ("2" "0" .. "3")
     mm = "0".."5" digit
 
-    delim = ","
+    newLine = ("\r\n" | "\n" | "\r")
+    delim =
+        | "," -- comma_delimiter
+        | "/" newLine "88," -- continuation_record_delimiter
```

Comment:
This is big. I have added support for `88 - Continuation Record` [pg. 20]. It is simplest to treat `\n88,` as a _delimiter_ rather than a new record.

Reply:
If a record is "split" **between** fields, it will look like
```
...field1,field2/
88,field3,field4...
```
which we want to parse as `field1,field2,field3,field4`

> The slash replaces the comma that ordinarily delimits that field. The example below correctly illustrates the use of delimiters.

![Screen Shot 2021-03-02 at 3 57 32 PM](https://user-images.githubusercontent.com/9813258/109715732-0082ea80-7b72-11eb-9c9a-04157686aec1.png)